tests/test_frontdoor.py

```diff
@@ -1374,6 +1374,7 @@ def test_on_check_page_back_button_goes_to_correct_location(has_loft_insulation)
     else:
         assert page.has_one("h1:contains('Do you have a loft that has not been converted into a room?')")
 
+
```

Comment:
Looks like a stray newline here

Reply:
The linter wants this here for check-code, should I still remove this?

Reply:
Oh I see, it was wrong before was it - no problem then